Configure

```diff
@@ -1650,6 +1652,16 @@ unless ($disabled{ktls}) {
 
 push @{$config{openssl_other_defines}}, "OPENSSL_NO_KTLS" if ($disabled{ktls});
 
+# TFO is support on some platforms, so just include only those
+unless ($disabled{tfo}) {
+    $config{tfo}="";
+    if (!($target =~ m/(^linux|^darwin|^BSD)/)) {
+	disable('not-supported', 'tfo');
```

Comment:
What happens on older linux/darwin/BSD platforms that don't support TCP Fast Open?

Reply:
I have other changes to handle that... but that being said, having it as a configuration option avoids any possible issues. It sounds as though both you and @levitte are against it being a configuration option... If that's the case, I can certainly change it.